### PR TITLE
docs: instructions on how to update the backports spreadsheet

### PR DESCRIPTION
**Pull request by Munkybooty**
This pr is made so that we have instructions on how to update the backports spreadsheet.

### CODE REVIEW

File: contrib/backports/backports-update.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+This allows you to easily cherry-pick merges and look into logs of bitcoin without switching directories.
+
+### Pulling Merges
+
+To pull the most up-to-date merges first make sure bitcoin is up-to-date:
+
+```
+git fetch bitcoin
```
Comment (PastaPastaPasta) [inline contrib/backports/backports-update.md:28]: nit, you already fetched bitcoin, could remove the one above

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

File: contrib/backports/backports-update.md (hunk 1 above)
Comment (Munkybooty) [reply contrib/backports/backports-update.md:28]: I left that in for the purpose of making sure that master is up to date if someone already has the remote added but not updated

**Review by PastaPastaPasta — DISMISSED** [review]
re-utACK

Not sure this is the correct path. Maybe doc/ makes more sense?

**Comment by UdjinM6** [comment]
>Not sure this is the correct path. Maybe doc/ makes more sense?

Let's just maybe make it another section after https://github.com/dashpay/dash/blob/develop/CONTRIBUTING.md#backporting instead of adding a file somewhere?

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
I don't really love this location either... 

IMO CONTRIBUTING.md should be for information virtually everyone contributing to the project needs to know. But, I don't think this falls under that category, but :shrug: 

utACK for squash merge

**Comment by Munkybooty** [comment]
I agree but I can't think of any future additions for an entire backporting directory that would be in the original place we kept it

**Pull request merged**